fragments/rhn-register.sh

```diff
@@ -44,7 +85,7 @@ subscription-manager repos \
                      --enable="rhel-7-server-rpms" \
                      --enable="rhel-7-server-extras-rpms" \
                      --enable="rhel-7-server-optional-rpms" \
-                     --enable="rhel-7-server-ose-3.2-rpms"
+                     --enable="rhel-7-server-ose-${OSE_VERSION}-rpms"
```

Comment:
@jprovaznik will this work with the shell variable enclosure around the name? If this is a template and the template string is `$OSE_VERSION` then `${OSE_VERSION}` won't match.   If these are set as environment variables then it will work correctly.